packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -212,18 +214,36 @@ const walk = async (dir: string) => {
 					);
 				}
 
-				// Each manifest entry is [header, pathHash, contentHash].
-				// In dev we want to avoid reading and hashing every file every time we
-				// reload the dev server, so in dev the 'content hash' is hash(path + modifiedTime).
-				// This way the 'content hash' still updates on file change, so the eTag also updates,
-				// and we don't incorrectly get a 304.
-				const pathHash = await hashPath(
-					encodeFilePath(relativeFilepath, path.sep)
-				);
-				const modifiedTime = (await fs.stat(filepath)).mtimeMs;
-				const contentHash = await hashPath(
-					encodeFilePath(relativeFilepath, path.sep) + modifiedTime.toString()
-				);
+				/*
+				 * Each manifest entry is a series of bytes that store data in a [header,
+				 * pathHash, contentHash] format, where:
+				 *   - `header` is a fixed ??x?? bytes reserved for ??
```

Comment:
I didn't have the exact data here, so you'll need to fill in `??x?? bytes` and what they are reserved for. Sorry abt that

Reply:
oops yes will do